src/CMakeLists.txt

```diff
@@ -351,6 +367,12 @@ endif(HAVE_GOOD_YASM_ELF64)
 add_library(common_texttable_obj OBJECT
   common/TextTable.cc)
 
+if(${LINUX})
```

Comment:
i'd recommend use `list(APPEND libcommon_files)` different files for different OS.